playbooks/roles/hammer_devel/tasks/hammer_install.yml

```diff
@@ -7,6 +7,12 @@
     remote: upstream
   with_items: "{{ hammer_devel_repositories }}"
 
+- name: 'Add fork remotes to cloned repositories'
```

Comment:
@ehelms I thought of that, but it seemed like quite a bit more work for not much reward. It's very unlikely that this command will fail, other than the remote already being created. Even if the remote is invalid (i.e. the url doesn't exist on GitHub), it will still pass. I'm willing to do what you propose if you can convince me there's a good reason to.